### PR TITLE
Local development using Docker environment

### PR DESCRIPTION
**Pull request by swaroopch**
I'm putting this up for review by @TheOriginalAlex & @anoff - I'm not certain this is correct w.r.t. building the local source code...

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
 build:
 	docker build -t openmined/pyyashe .
 
+# Run the source code inside the Docker environment
 run:
 	docker run -i -t openmined/pyyashe python3 -c "import example; example.test()"
+
+# Run *my* source code with the Docker environment
+dev:
+	docker run -i -t -v "$(PWD):/PyYashe" openmined/pyyashe sh run.sh
```
Comment (ghost) [inline Makefile]: This issue with this is that the build actually runs the python setup which builds the library and installs it.  Mounting the code isn't gonna reflect the changes unless you build it again.

**Review by anoff — APPROVED** [review]
Lgtm. Great idea to put all commands in the makefile! If someone can independently validate that it works for them merge ahead! Don't have access to my machine atm

**Comment by swaroopch** [comment]
@mayank0403 Can you please confirm that this branch works for you locally?

**Comment by swaroopch** [comment]
@anoff Thanks for your review, will wait for @mayank0403 's confirmation that it works locally.

File: Makefile (hunk 1 above)
Comment (swaroopch) [reply Makefile]: @TheOriginalAlex Can you please check the latest revision, I have changed it to compile and run. (In the latest revision, I have removed the shell scripts, everything is in the Makefile now)

Comment (ghost) [reply Makefile]: Ah.  Yea.  The latest stuff looks good to me!  :)

Comment (ghost) [reply Makefile]: I think I wrote the comment a few days ago, but forgot to submit the review.

Comment (swaroopch) [reply Makefile]: @TheOriginalAlex Thanks for confirming :)

**Pull request merged**